lib/elements/polyline.py

```diff
@@ -32,6 +32,13 @@ class Polyline(EmbroideryElement):
     # users use File -> Import to pull in existing designs they may have
     # obtained, for example purchased fonts.
 
+    element_name = "Polyline"
+
+    @property
+    @param('satin_column', _('Satin stitch along paths'), type='toggle', inverse=True)
```

Comment:
Why do we need this?  I think I can guess but I'm not sure.

Reply:
Because params would give us an error on polylines. It shouldn't be called satin column though ... it's only where I stole it from :D I will have to rename it.

It doesn't make a lot of sense here, because we have not really any option to set, but at least it indicates the object type to the user when he calls params on it and prevents the error message.

Reply:
Yeah, I get it now.  This is all down to my somewhat questionable code design for params. :)

I'm hoping to revamp this when we convert to Electron.  The `@param` decorators are used to generate the UI, which is kind of neat from a code perspective, but it really limits us in the UI.  Perhaps we should ditch that when we reimplement in Vue.js?

Reply:
Hm, yes, we maybe should. We will be very busy ;)